include/zephyr/bluetooth/conn.h

```diff
@@ -1247,7 +1247,8 @@ struct bt_conn_cb {
 				const struct bt_conn_le_path_loss_threshold_report *report);
 #endif /* CONFIG_BT_PATH_LOSS_MONITORING */
 
-	struct bt_conn_cb *_next;
+	/** Internally used field for list handling */
+	sys_snode_t node;
```

Comment:
Should we name it 
```suggestion
	/** Internally used field for list handling */
	sys_snode_t _node;
```
To further indicate that this is internal? 

Does the doxygen tag `@internal` work for fields in structs?

Reply:
I will give it a try